elex/__init__.py

```diff
@@ -6,16 +6,22 @@
 from cachecontrol.caches import FileCache
 from elex.cachecontrol_heuristics import EtagOnlyCache
 
-__version__ = '2.4.3'
-_DEFAULT_CACHE_DIRECTORY = os.path.join(tempfile.gettempdir(), 'elex-cache')
+__version__ = "2.4.3"
```

Comment:
@vutrobinnj I'm thinking we should increment the minor version number to `2.4.4` here. That could also help avoid unwanted caching by `pip`.

(We have one that when making functional changes to other third-party repos we've forked, ex. `vue-treeselect`.)

There won't be any conflicts with versioning in the original upstream repo because the original one is frozen at `2.4.3`.